guides/plugins/plugins/administration/using-base-components.md

```diff
@@ -0,0 +1,37 @@
+# Using base components
+
+The Shopware 6 Administration registers all components into the component registry wich makes them available in the entire Administration
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/), the framework, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Finding the base component needed
+
+All Shopware 6 Administration components can be seen in the [Component Library](https://component-library.shopware.com/). 
+There you can see what each of the components does and looks like, it also shows you what props the can work with and wich slots they have.
+
+## Using the base
+
+As mentioned before in the introduction all components used in the Shopware 6 Administration are registered in the `component registry`, which makes them accessible everywhere.
+Learn what the `component registry` is and how you can register your own components to it [here](./add-custom-component.md)
+
+In this example we will use the `sw-text-field` in our template.
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/example-component/example.html.twig" %}
+```html
+<div>
+    <sw-text-field />
```

Comment:
This example and the whole paragraph feels rather short and abrupt. I already tries to improve that thorough using more text. Could you use an example closer to the real world?

Reply:
Well then it would proably overlap to much with the [writing templates](https://github.com/shopware/docs/blob/master/guides/plugins/plugins/administration/writing-templates.md) article, wich itself already overlaps with the [add-custom-component](https://github.com/shopware/docs/blob/master/guides/plugins/plugins/administration/add-custom-component.md) article

Reply:
I know, that's tough 🙈  Maybe using a bigger example then? With some probs handled into `sw-text-field` or something? 🙈